tests/flows.spec.ts

```diff
@@ -57,7 +58,178 @@ test('Applicant should be able to fill the form with a browser, submit documents
 
   await submittingDocuments(desktopPage);
   await expect(desktopPage.getByRole('heading', { name: /Verification/i })).toHaveText('Verification failed');
+  await desktopPage.screenshot({ path: 'tests/screenshots/failed.png', fullPage: true });
 
   await desktopPage.getByRole('link', { name: 'Try again' }).click();
   await expect(desktopPage.getByText(/Verify your identity/i)).toHaveText('Verify your identity');
 });
+
+test('Applicant should be able to fill the form with a browser, submit documents and photo with a phone and see a document not supported error message', async () => {
+  const desktopPage = await desktop.newPage();
+
+  await fillStartForm(desktopPage, { ...applicant, firstName: 'Image Integrity - Supported Document' });
+  const url = await continueOnfidoFlowThenGetAndTestLink(desktopPage, expect);
+
+  const mobilePage = await mobile.newPage();
+  await openKycLinkAndTestDocumentAndPhotoScan(url, mobilePage, expect);
+
+  await submittingDocuments(desktopPage);
+  await expect(await desktopPage.getByRole('list', { name: 'error list' })).toHaveText('The document provided is not supported');
+  await expect(desktopPage.getByText(/We could not verify your identity/)).toHaveText('We could not verify your identity. We invite you to read the reasons below and try again.');
+  await desktopPage.screenshot({ path: 'tests/screenshots/failed_image_integrity_supported_docs.png', fullPage: true });
+});
```

Comment:
> 'Applicant should be able to fill the form with a browser, submit documents and photo with a phone and see a document not supported error message'

Really it's more like 'Upon submitting an unsupported document, applicant should see an error message about the document not being supported'.

Within this test, it's currently hard for me to see where you're intentionally submitting an unsupported document. 

I think ideally for each of the tests it would be easier to see why you're expecting error messages.

Reply:
Thanks, I rewrote most of the sentences.